### PR TITLE
Update data wrangling

### PR DESCRIPTION
**Pull request by mariarrt94**
Update data wrangling: 

- Added detail about here
- Added information about tidyverse functions
- Missing: add the usethis::use_course()

### CODE REVIEW

File: Presentations/03-data-wrangling.Rmd [hunk 1]
```diff
@@ -98,17 +98,17 @@ name: intro
 .panelset[
 
 .panel[.panel-name[If You Attended Session 2]
-1. Go to the `dime-r-training-mar2023` folder that you created yesterday, and open the `dime-r-training-mar2023` R project that you created there.
+1. Go to the `dime-r-training-mar2024` folder that you created yesterday, and open the `dime-r-training-mar2024` R project that you created there.
 ]
 
 .panel[.panel-name[If You Did Not Attend Session 2]
-1. Create a folder named `dime-r-training-mar2023` in your preferred location in your computer.
+1. Create a folder named `dime-r-training-mar2024` in your preferred location in your computer.
```
Comment (luisesanmartin) [inline Presentations/03-data-wrangling.Rmd:105]: this is the tab that should have the instructions with `use_course()`

File: Presentations/03-data-wrangling.Rmd [hunk 2]
```diff
@@ -605,7 +666,7 @@ name: variables
 
 # Creating new variables
 
-We use `mutate()` to create new variables. For example:
+We use `dplyr::mutate()` to create new variables. For example:
```
Comment (luisesanmartin) [inline Presentations/03-data-wrangling.Rmd:669]: make sure to explain that the `::` notation means that you're using a function from a package

**Review by luisesanmartin — COMMENTED** [review]
One more comment: the part about packages is also a recap, they're introduced in the previous session.

Also, a comment for the presentation: Rony and Marc-Andrea usually switched windows between the presentation anr RStudio for this session. I'm personally not a fan of that, my sense is that it can be a bit confusing because of the latency between the presenter's computer screen and what the attendants see in their Windows and because that gives less time for them to see the slides, but feel free to choose if that's how you want to complement the slides when presenting or if only presenting slides is okay. In any case, if you choose to run code in RStudio when presenting, try to not change the windows suddenly (I think it's better to say that you're changing a window) and make sure to explain how what you're doing related to the slides

File: Presentations/03-data-wrangling.Rmd (hunk 1 above)
Comment (mariarrt94) [reply Presentations/03-data-wrangling.Rmd]: It is added now

File: Presentations/03-data-wrangling.Rmd (hunk 2 above)
Comment (mariarrt94) [reply Presentations/03-data-wrangling.Rmd:697]: Good point!  I will

**Comment by mfiorina** [comment]
Review of the slides:
- Slide 5: I'd suggest replacing "To organize data in a way that it will be easier to analyze and communicate it." with "To organize data in a manner that makes it easier to analyze and communicate."
- Slide 7: "R packages are apps in your phone" should be "on your phone"
- Slide 8: recommend using the same package example for the use of `install_packages()` and `library()`, either `dplyr` or `tidyverse`
- Slide 9: there is a warning in the HTML (`Warning: package 'janitor' was built under R version 4.3.1`) that could be resolved by updating R prior to running the .Rmd file. Might be worth doing to avoid extra questions.
- Slide 14: typo in bottom text, "want" instead of "wan"
- Slide 42: would recommend at least orally mentioning the existence of `right_join()` and `full_join()` for context on how`left_join()` works

That's it, hope this helps!

**Pull request merged**